src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,11 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+        return 2 * (Math.sin(x - Math.PI / 6)) * calculateB(z) / (0.5 + Math.pow(Math.sin(y), 2));
```

Comment:
This code is unreadable. I'd prefer to extract some logic into meaningful variables.